src/main/java/net/consensys/wittgenstein/core/Node.java

```diff
@@ -37,6 +40,7 @@
   protected long msgSent = 0;
   protected long bytesSent = 0;
   protected long bytesReceived = 0;
+  public final Optional<String> cityName;
```

Comment:
I think it's safer to have a global city by default, this save the optional part.
The current code does not use optional, some fields can be null, but optional is never used. For consistency it's better not use optional (or then do a global change)...

Reply:
I will change it to default value